plugins/toolkit/core/build.gradle.kts

```diff
@@ -23,7 +23,7 @@ dependencies {
 //    api(libs.aws.ssooidc)
     api(libs.aws.sts)
     api(libs.bundles.jackson)
-
+    implementation(libs.commonmark)
```

Comment:
There is no other usage of commonmark so we can remove the dependency from `toolkit/jetbrains-core`

Reply:
I will move `commonmark` and its related usecase from `toolkit/jetbrains-core` to `core `.